### PR TITLE
docs: fix example typo

### PR DESCRIPTION
**Pull request by lejenome**
---
title: "fix example typo"
---

Fix a typo on the "Channel Item Object Example" YAML example in the specification file.

### CODE REVIEW

**Review by smoya — APPROVED** [review]
LGTM!, even though you will need a +1 from a Code Owner.

**Review by dalelane — APPROVED** [review]

**Review by derberg — APPROVED** [review]

**Comment by derberg** [comment]
@all-contributors please add @lejenome for docs

**Pull request merged**
